lib/collection/src/common/sha_256.rs

```diff
@@ -25,6 +41,50 @@ pub async fn hash_file(file_path: &Path) -> io::Result<String> {
     Ok(format!("{hash:x}"))
 }
 
+/// Compute sha256 hash for the given file using the `sha256sum` binary
+///
+/// May fail if the binary does not exist on the system.
+#[cfg(all(debug_assertions, target_os = "linux"))]
+async fn hash_file_bin(file_path: &Path) -> io::Result<String> {
+    use tokio::process::Command;
+
+    let output = match Command::new("sha256sum")
+        .arg("-b")
+        .arg(file_path)
```

Comment:
Security/correctness: prepend `--` argument when passing arbitrary filenames so weird filenames like `--help` won't be treated as options.

```suggestion
    let output = match Command::new("sha256sum")
        .arg("-b")
        .arg("--")
        .arg(file_path)
```

Reply:
Thanks. Didn't think about that one!